xayn-ai/src/utils.rs

```diff
@@ -78,6 +78,12 @@ pub(crate) fn mock_uuid(sub_id: usize) -> uuid::Uuid {
     uuid::Uuid::from_u128(BASE_UUID | (sub_id as u128))
 }
 
+/// Creates a CoI id from a mock UUID.
+#[cfg(test)]
+pub(crate) fn mock_coiid(sub_id: usize) -> crate::CoiId {
```

Comment:
Minor nitpicking: I think it should be `mock_coi_id` given that it's also `CoidId` and not `Coiid`.